src/views/job-role-list.njk

```diff
@@ -129,4 +212,7 @@
     {% endif %}
 </div>
 </main>
+
+<script src="/js/filterJobRoles.js"></script>
```

Comment:
I think filtering of job roles should be handled by a controller rather than an imported script. 

If we disabled JS in the browser this wouldn't work.